_docs/tasks/telemetry/servicegraph.md

```diff
@@ -77,35 +77,57 @@ the example application throughout this task.
    kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=servicegraph -o jsonpath='{.items[0].metadata.name}') 8088:8088 &   
    ```
 
-   Visit [http://localhost:8088/dotviz](http://localhost:8088/dotviz) in your web browser.
+   Visit
+   [http://localhost:8088/force/forcegraph.html](http://localhost:8088/force/forcegraph.html)
+   in your web browser. Try clicking on a service to see details on
+   the service. Real time traffic data is shown in a panel below.
 
    The results will look similar to:
 
-   {% include figure.html width='100%' ratio='63.16%'
+   {% include figure.html width='75%' ratio='100%'
```

Comment:
The ratio needs to reflect the true ratio of your image, otherwise it'll show up squashed. From the docs:

You need to fill in all the values. The width represents the percentage of space used by the image relative to the surrounding text. The ratio is (image height / image width) * 100.